### ISSUE TITLE
Add Preview Tab for GitHub Documentation Feedback

### ISSUE THREAD

**Issue by docs-product** [MicrosoftDocs/feedback#11]
When I'm creating issues on docs.microsoft.com does all of markdown goodness work? After submitting another issue, I now see it does but it is not obvious. Where's my preview tab that renders my markdown?!

https://github.com/MicrosoftDocs/TeamBlogFeedback/issues/1

----------
⚠ Idea migrated from UserVoice

**Created By:** Mike DePouw
**Created On:** 2018/03/01 19:15:48 +0000
**Votes at Migration:** 3
**Supporters at Migration:** 3

**Comment by dend** [comment]
@ShevaDas as the experience PM - FYI.

**Comment by ShevaDas** [comment]
Thanks for the feedback! We don't have plans to add a preview tab currently but we'll take this into consideration for future work.

**Comment by ShevaDas** [comment]
I've filed an issue for this in our internal work tracker and will follow up when it's implemented.

**Comment by spottedmahn** [comment]
>will follow up when it's implemented

Does this mean you guys plan to implement it?

**Comment by ShevaDas** [comment]
@spottedmahn It's something we're looking into. @jialge is the point of contact for this work now and can speak to further timelines and implementation.

**Comment by mlottner** [comment]
Closing this issue now. Feel free to re-open if/as you see a continued need.

**Comment by spottedmahn** [comment]
>Closing this issue now

😢.  Also, I can't reopen it b/c of the migration from UserVoice 😉

>It's something we're looking into

is that no longer the case?

**Comment by mlottner** [comment]
@spottedmahn We're investigating it, but leaving issues endlessly open is not the ideal workflow for GitHub feedback. Closing a feedback issue (as I did here) simply means we've assigned an internal owner and the conversation will/can continue. It allows us to continue working to improve your experience without having an unmanageable queue of "open" issues in the backlog. Absolutely feel free to keep discussing/tracking it here.
